Error count in import log shows total warnings

## Bug Report 

##### SUMMARY
If you take a look at [this test import in QA](https://galaxy-qa.ansible.com/my-imports/5668), you'll notice that the total number of warnings is 0, while the total number of errors is 128. However, scroll through the log, and notice there are no errors, only warnings. 

**Note**: Full log captured below under Actual Results

##### STEPS TO REPRODUCE
1. clone [github.com/bcoca/collection](https://github.com/bcoca/collection)
2. update `galaxy.yml`
3. build the collection
4. publish the collection

##### EXPECTED RESULTS
The last line of the output should read: `Import completed with 128 warnings and 0 errors `

##### ACTUAL RESULTS

```
Starting import: task_id=10, artifact_id=10 
===== LOADING ROLE ===== 
Role in collection gets name from directory, ignoring the 'role_name' attribute 
Checking role metadata tags 
Checking role platforms 
Checking role cloud platforms 
Checking role dependencies 
 
===== LINTING ROLE: minishift-up-role ===== 
yamllint OK. 
ansible-lint Warnings: 
./roles/minishift-up-role/meta/main.yml:2: [E201] Trailing whitespace 
./roles/minishift-up-role/meta/main.yml:4: [E201] Trailing whitespace 
./roles/minishift-up-role/meta/main.yml:18: [E201] Trailing whitespace 
./roles/minishift-up-role/meta/main.yml:20: [E201] Trailing whitespace 
./roles/minishift-up-role/meta/main.yml:45: [E201] Trailing whitespace 
./roles/minishift-up-role/meta/main.yml:47: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/Darwin_driver.yml:3: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/Darwin_driver.yml:7: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/Darwin_driver.yml:8: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/Darwin_driver.yml:11: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/Darwin_driver.yml:13: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/Darwin_driver.yml:17: [E302] chown used in place of argument owner to file module 
./roles/minishift-up-role/tasks/Darwin_driver.yml:21: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/Darwin_driver.yml:23: [E302] chmod used in place of argument mode to file module 
./roles/minishift-up-role/tasks/Darwin_driver.yml:24: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/Darwin_driver.yml:27: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/Debian_driver.yml:5: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/Debian_driver.yml:7: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/Debian_driver.yml:10: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/Debian_driver.yml:19: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/Debian_driver.yml:21: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/Debian_driver.yml:27: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/Debian_driver.yml:30: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/Debian_driver.yml:30: [E305] Use shell only when shell functionality is required 
./roles/minishift-up-role/tasks/Debian_driver.yml:32: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/RedHat_driver.yml:10: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/RedHat_driver.yml:16: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/RedHat_driver.yml:19: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/RedHat_driver.yml:21: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/RedHat_driver.yml:27: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/RedHat_driver.yml:30: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/RedHat_driver.yml:30: [E305] Use shell only when shell functionality is required 
./roles/minishift-up-role/tasks/RedHat_driver.yml:31: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/add_hostname.yml:1: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/add_hostname.yml:8: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/add_hostname.yml:9: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/add_hostname.yml:11: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/cluster_up.yml:6: [E502] All tasks should be named 
./roles/minishift-up-role/tasks/cluster_up.yml:7: [E601] Don't compare to literal True/False 
./roles/minishift-up-role/tasks/cluster_up.yml:15: [E502] All tasks should be named 
./roles/minishift-up-role/tasks/cluster_up.yml:16: [E601] Don't compare to literal True/False 
./roles/minishift-up-role/tasks/copy_oc_client.yml:21: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/create_objects.yml:1: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/create_objects.yml:1: [E303] mktemp used in place of tempfile module 
./roles/minishift-up-role/tasks/create_objects.yml:9: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/create_objects.yml:10: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/create_objects.yml:12: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/create_objects.yml:14: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/create_objects.yml:15: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/create_objects.yml:17: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/get_ms_assets.yml:7: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/get_ms_assets.yml:7: [E602] Don't compare to empty string 
./roles/minishift-up-role/tasks/get_ms_assets.yml:12: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/get_ms_assets.yml:12: [E602] Don't compare to empty string 
./roles/minishift-up-role/tasks/get_ms_assets.yml:18: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/get_ms_assets.yml:18: [E602] Don't compare to empty string 
./roles/minishift-up-role/tasks/get_ms_assets.yml:23: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/get_ms_assets.yml:23: [E602] Don't compare to empty string 
./roles/minishift-up-role/tasks/get_ms_darwin_asset.yml:1: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/get_ms_linux_asset.yml:1: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/grant_admin.yml:1: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/grant_admin.yml:1: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/grant_admin.yml:4: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/grant_admin.yml:4: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/grant_admin.yml:7: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/grant_admin.yml:9: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/grant_admin.yml:10: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/grant_admin.yml:11: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/install_minishift.yml:3: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/install_minishift.yml:17: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/install_minishift.yml:17: [E303] mktemp used in place of tempfile module 
./roles/minishift-up-role/tasks/install_minishift.yml:24: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/install_minishift.yml:26: [E303] unzip used in place of unarchive module 
./roles/minishift-up-role/tasks/install_minishift.yml:27: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/install_minishift.yml:32: [E303] tar used in place of unarchive module 
./roles/minishift-up-role/tasks/install_minishift.yml:33: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/install_minishift.yml:38: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/install_minishift.yml:38: [E306] Shells that use pipes should set the pipefail option 
./roles/minishift-up-role/tasks/install_minishift.yml:39: [E206] Variables should have spaces before and after: {{ var_name }} 
./roles/minishift-up-role/tasks/install_minishift.yml:47: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/isrunning.yml:1: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/isrunning.yml:12: [E601] Don't compare to literal True/False 
./roles/minishift-up-role/tasks/isrunning.yml:30: [E201] Trailing whitespace 
./roles/minishift-up-role/tasks/main.yml:23: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/main.yml:30: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/main.yml:30: [E306] Shells that use pipes should set the pipefail option 
./roles/minishift-up-role/tasks/main.yml:48: [E301] Commands should not change things if nothing needs doing 
./roles/minishift-up-role/tasks/main.yml:48: [E306] Shells that use pipes should set the pipefail option 
 
===== IMPORTING ROLE: minishift-up-role ===== 
 
===== LOADING MODULE ===== 
 
===== LINTING MODULE: k8s_scale ===== 
flake8 OK. 
 
===== IMPORTING MODULE: k8s_scale ===== 
 
===== LOADING MODULE ===== 
 
===== LINTING MODULE: k8s_service ===== 
flake8 Warnings: 
plugins/modules/k8s_service.py:64:121: E501 line too long (140 > 120 characters) 
plugins/modules/k8s_service.py:67:121: E501 line too long (123 > 120 characters) 
plugins/modules/k8s_service.py:170:1: F401 'ansible.module_utils.k8s.common.COMMON_ARG_SPEC' imported but unused 
 
===== IMPORTING MODULE: k8s_service ===== 
 
===== LOADING MODULE ===== 
 
===== LINTING MODULE: k8s_facts ===== 
flake8 OK. 
 
===== IMPORTING MODULE: k8s_facts ===== 
 
===== LOADING MODULE ===== 
 
===== LINTING MODULE: k8s ===== 
flake8 Warnings: 
plugins/modules/k8s.py:50:121: E501 line too long (140 > 120 characters) 
plugins/modules/k8s.py:53:121: E501 line too long (123 > 120 characters) 
plugins/modules/k8s.py:64:121: E501 line too long (128 > 120 characters) 
plugins/modules/k8s.py:66:121: E501 line too long (130 > 120 characters) 
plugins/modules/k8s.py:82:121: E501 line too long (124 > 120 characters) 
plugins/modules/k8s.py:83:121: E501 line too long (121 > 120 characters) 
plugins/modules/k8s.py:84:121: E501 line too long (140 > 120 characters) 
plugins/modules/k8s.py:97:121: E501 line too long (121 > 120 characters) 
plugins/modules/k8s.py:98:121: E501 line too long (141 > 120 characters) 
plugins/modules/k8s.py:122:121: E501 line too long (131 > 120 characters) 
plugins/modules/k8s.py:123:121: E501 line too long (129 > 120 characters) 
 
===== IMPORTING MODULE: k8s ===== 
 
===== LOADING MODULE ===== 
 
===== LINTING MODULE: k8s_auth ===== 
flake8 Warnings: 
plugins/modules/k8s_auth.py:202:121: E501 line too long (122 > 120 characters) 
plugins/modules/k8s_auth.py:254:9: F841 local variable 'e' is assigned to but never used 
plugins/modules/k8s_auth.py:314:9: F841 local variable 'ret' is assigned to but never used 
 
===== IMPORTING MODULE: k8s_auth ===== 
 
===== LOADING MODULE_UTILS ===== 
 
===== LINTING MODULE_UTILS: raw ===== 
flake8 Warnings: 
plugins/module_utils/raw.py:34:1: F811 redefinition of unused 'LooseVersion' from line 23 
plugins/module_utils/raw.py:39:121: E501 line too long (133 > 120 characters) 
plugins/module_utils/raw.py:45:5: F401 'kubernetes_validate' imported but unused 
plugins/module_utils/raw.py:147:121: E501 line too long (139 > 120 characters) 
plugins/module_utils/raw.py:194:121: E501 line too long (140 > 120 characters) 
plugins/module_utils/raw.py:285:121: E501 line too long (135 > 120 characters) 
plugins/module_utils/raw.py:295:121: E501 line too long (139 > 120 characters) 
plugins/module_utils/raw.py:310:121: E501 line too long (134 > 120 characters) 
plugins/module_utils/raw.py:320:121: E501 line too long (139 > 120 characters) 
plugins/module_utils/raw.py:348:121: E501 line too long (135 > 120 characters) 
plugins/module_utils/raw.py:418:82: W504 line break after binary operator 
plugins/module_utils/raw.py:419:87: W504 line break after binary operator 
plugins/module_utils/raw.py:423:77: W504 line break after binary operator 
plugins/module_utils/raw.py:427:94: W504 line break after binary operator 
plugins/module_utils/raw.py:428:93: W504 line break after binary operator 
plugins/module_utils/raw.py:463:121: E501 line too long (139 > 120 characters) 
 
===== IMPORTING MODULE_UTILS: raw ===== 
 
===== LOADING MODULE_UTILS ===== 
 
===== LINTING MODULE_UTILS: common ===== 
flake8 Warnings: 
plugins/module_utils/common.py:21:1: F401 'json' imported but unused 
plugins/module_utils/common.py:197:121: E501 line too long (139 > 120 characters) 
plugins/module_utils/common.py:199:121: E501 line too long (121 > 120 characters) 
 
===== IMPORTING MODULE_UTILS: common ===== 
 
===== LOADING MODULE_UTILS ===== 
 
===== LINTING MODULE_UTILS: scale ===== 
flake8 Warnings: 
plugins/module_utils/scale.py:32:1: F841 local variable 'exc' is assigned to but never used 
 
===== IMPORTING MODULE_UTILS: scale ===== 
 
===== LOADING FILTER_PLUGIN ===== 
 
===== LINTING FILTER_PLUGIN: k8s ===== 
flake8 OK. 
 
===== IMPORTING FILTER_PLUGIN: k8s ===== 
 
===== LOADING INVENTORY_PLUGIN ===== 
 
===== LINTING INVENTORY_PLUGIN: k8s ===== 
flake8 Warnings: 
plugins/inventory/k8s.py:158:121: E501 line too long (137 > 120 characters) 
plugins/inventory/k8s.py:319:13: F841 local variable 'e' is assigned to but never used 
 
===== IMPORTING INVENTORY_PLUGIN: k8s ===== 
 
Import completed with 0 warnings and 128 errors 
```

--- a/galaxy/worker/tasks/collection.py
+++ b/galaxy/worker/tasks/collection.py
@@ -59,7 +59,7 @@ def import_collection(artifact_id, repository_id):
         task_logger.error(f'Import Task "{task.id}" failed: {e}')
         raise
 
-    warnings, errors = task.get_message_stats()
+    errors, warnings = task.get_message_stats()
     task_logger.info(
         f'Import completed with {warnings} warnings and {errors} errors')
 

